File: Python/8py 1/between.py
"""
|--------------------------------------------------------------------------
| What is between?
|--------------------------------------------------------------------------
|
"""

def between(a,b):
   return [i for i in range(a,b+1)]


def between(a,b):
   res = []
   while a <= b:
      res.append(a)
      a += 1
   return res


def between(a,b):
   return list(range(a,b+1))


print(between(1, 4), [1, 2, 3, 4])
print(between(-2, 2), [-2, -1, 0, 1, 2])

File: PHP/8php 1/between.php
<?php
/*
|--------------------------------------------------------------------------
| What is between?
|--------------------------------------------------------------------------
|
*/

function between(int $a, int $b): array {
   return range($a, $b);
}


function between(int $a, int $b): array {
   $res = [];
   for ($i = $a; $i <= $b; $i++) {
      $res[] += $i;
   }
   return $res;
}


function between(int $a, int $b): array {
   $arr = [];
   for(; $a<=$b; $a++) array_push($arr, $a);
   return $arr;
}


print_r(between(1, 4));
print_r(between(-2, 2));